docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc

```diff
@@ -0,0 +1,58 @@
+[role="xpack"]
+[[ml-dfa-limitations]]
+== {dfanalytics-cap} limitations
+[subs="attributes"]
+++++
+<titleabbrev>Limitations</titleabbrev>
+++++
+
+experimental[]
+
+The following limitations and known problems apply to the 7.3 release of 
+the Elastic {dfanalytics} feature:
+
+[float]
+[[dfa-ccs-limitations]]
+=== {ccs-cap} limitation
+
+{ccs-cap} is not supported in 7.3 for {dfanalytics}.
+
+[float]
+[[dfa-deletion-limitations]]
+=== Deleting a {dfanalytics-job} does not delete the {dataframe} destination index
+
+When deleting a {dfanalytics-job} using 
+`DELETE _ml/data_frame/analytics/<data_frame_analytics_id>` the {dataframe} 
+destination index won't be deleted. This object must be deleted separately.
+
+[float]
+[[dfa-update-limitations]]
+=== Cannot update a {dfanalytics-job}
+
+{dfanalytics-cap} configurations cannot be updated. Please delete and 
```

Comment:
Is there a reason why you must delete it or is that optional (as long as your new job has a new name)?

Reply:
AFAIK, it is optional, you don't need to delete it if you are okay with another name for the job.

Reply:
The only reason to delete is if for some reason you really want the job to have that specific `id`. Otherwise you can simply create a new job and the fate of the old one is independent :-)